tests/Fixtures/Integration/priority/comment_to_phpdoc,phpdoc_to_comment.test

```diff
@@ -1,31 +0,0 @@
---TEST--
-Integration of fixers: comment_to_phpdoc,phpdoc_to_comment.
---RULESET--
-{"comment_to_phpdoc": true, "phpdoc_to_comment": true}
```

Comment:
no more priority conflicts between them ?

Reply:
exactly, I could update this to switching comment types test, but that would not be priority test and `CommentsAnalyzer::isPhpdocCandidate` makes it pretty easy to see they not conflicting

Reply:
please restore the test, simply move it to `misc` from `priority`